src/main/java/org/agbrothers/englishbot/processing/processor/dictionary/AddWordDictionaryProcessor.java

```diff
@@ -43,9 +43,10 @@ public void setDictionaryService(DictionaryService dictionaryService) {
      * @return validation error message, {@code null} if validation succeeded
      */
     private String validateWordData(String[] wordData) {
-        if(isValidWordData(wordData)){
+        if(isValidWordData(wordData)) {
             return "Помилка в слові англійською або перекладі слова. " +
-                    "Слово англійською має бути латиницею, переклад - лише кирилицею.";
+                    "Слово англійською має бути латиницею, переклад - лише кирилицею. " +
+                    "Слова вводити малими літерами.";
```

Comment:
You should not constraint a user, you should allow the input regardless of the letter case and store it in DB in lowercase